mindmeld/text_preparation/text_preparation_pipeline.py

```diff
@@ -548,3 +565,49 @@ def create_default_text_preparation_pipeline():
         return TextPreparationPipelineFactory.create_text_preparation_pipeline(
             **DEFAULT_EN_TEXT_PREPARATION_CONFIG
         )
+
+    @staticmethod
+    def _instantiate_pipeline_components(  # pylint: disable=W0640
```

Comment:
Can we test this function ie pass in varied type classes and make sure it works as expected?

Reply:
Sure, added tests for valid and invalid input